server/accounts.go

```diff
@@ -3153,7 +3157,12 @@ func (s *Server) updateAccountClaimsWithRefresh(a *Account, ac *jwt.AccountClaim
 			c.mu.Lock()
 			sk := c.user.SigningKey
 			c.mu.Unlock()
-			if sk != "" && !a.hasIssuer(sk) {
+			if sk == "" {
```

Comment:
Maybe use `_EMPTY_` here, just to be consistent with most of code checking for empty string..

Reply:
fixed